[2025-09-02T02:01:04.227+0200] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fraud_detection_pipeline.check_data scheduled__2025-09-01T00:00:00+00:00 [queued]>
[2025-09-02T02:01:04.231+0200] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fraud_detection_pipeline.check_data scheduled__2025-09-01T00:00:00+00:00 [queued]>
[2025-09-02T02:01:04.231+0200] {taskinstance.py:1361} INFO - Starting attempt 1 of 3
[2025-09-02T02:01:04.237+0200] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): check_data> on 2025-09-01 00:00:00+00:00
[2025-09-02T02:01:04.248+0200] {standard_task_runner.py:57} INFO - Started process 96423 to run task
[2025-09-02T02:01:04.252+0200] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'fraud_detection_pipeline', 'check_data', 'scheduled__2025-09-01T00:00:00+00:00', '--job-id', '51', '--raw', '--subdir', 'DAGS_FOLDER/fraud_pipeline.py', '--cfg-path', '/var/folders/jw/3gjd6ln50hq0ww6wh8twy1780000gn/T/tmp1yrmn2_x']
[2025-09-02T02:01:04.253+0200] {standard_task_runner.py:85} INFO - Job 51: Subtask check_data
[2025-09-02T02:01:04.287+0200] {task_command.py:416} INFO - Running <TaskInstance: fraud_detection_pipeline.check_data scheduled__2025-09-01T00:00:00+00:00 [running]> on host macbookpro.home
[2025-09-02T02:01:04.328+0200] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data-science-team' AIRFLOW_CTX_DAG_ID='fraud_detection_pipeline' AIRFLOW_CTX_TASK_ID='check_data' AIRFLOW_CTX_EXECUTION_DATE='2025-09-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-01T00:00:00+00:00'
[2025-09-02T02:01:04.331+0200] {logging_mixin.py:154} WARNING - 2025-09-02 02:01:04,330 - fraud_detection - INFO - üîç D√©marrage de la v√©rification des donn√©es...
[2025-09-02T02:01:04.330+0200] {fraud_pipeline.py:94} INFO - üîç D√©marrage de la v√©rification des donn√©es...
[2025-09-02T02:01:04.334+0200] {logging_mixin.py:154} WARNING - 2025-09-02 02:01:04,333 - fraud_detection - INFO - Chargement des donn√©es depuis: /Users/adelinerobinet/Documents/Projets Python/exam-fluzz-fraud-detection/adeline/data/raw/creditcard.csv
[2025-09-02T02:01:04.333+0200] {fraud_pipeline.py:104} INFO - Chargement des donn√©es depuis: /Users/adelinerobinet/Documents/Projets Python/exam-fluzz-fraud-detection/adeline/data/raw/creditcard.csv
[2025-09-02T02:01:05.628+0200] {logging_mixin.py:154} WARNING - 2025-09-02 02:01:05,626 - fraud_detection - INFO - ‚úÖ Validation termin√©e: 284807 lignes, 0.173% fraudes
[2025-09-02T02:01:05.626+0200] {fraud_pipeline.py:150} INFO - ‚úÖ Validation termin√©e: 284807 lignes, 0.173% fraudes
[2025-09-02T02:01:05.631+0200] {python.py:194} INFO - Done. Returned value was: {'status': 'success', 'message': 'Dataset valide', 'data_quality': {'rows_count': 284807, 'columns_count': 31, 'missing_percentage': 0.0, 'fraud_percentage': 0.1727485630620034, 'imbalance_ratio': 577.8760162601626}, 'business_validation': {'time_range_valid': True, 'amount_range_valid': True, 'class_values_valid': True}, 'class_distribution': {0: 284315, 1: 492}}
[2025-09-02T02:01:05.646+0200] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=fraud_detection_pipeline, task_id=check_data, execution_date=20250901T000000, start_date=20250902T000104, end_date=20250902T000105
[2025-09-02T02:01:05.709+0200] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-09-02T02:01:05.724+0200] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
